Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -17,10 +17,18 @@ static QString GetSettingsPath()
   return QString::fromStdString(File::GetUserPath(D_CONFIG_IDX)) + QStringLiteral("/UI.ini");
 }
 
-Settings::Settings(QObject* parent) : QSettings(GetSettingsPath(), QSettings::IniFormat, parent)
+Settings::Settings() : QSettings(GetSettingsPath(), QSettings::IniFormat)
 {
 }
 
+Settings& Settings::instance()
+{
+  static Settings* s_settings = nullptr;
```

Comment:
By the way, if you want to make this thread-safe, you would put the initialization portion within a [std::call_once](http://en.cppreference.com/w/cpp/thread/call_once) call, so it's guaranteed to only ever be initialized by one thread.

alternativly, (and preferably, imho), just using a function local static would also be better.

i.e.

```cpp
Settings& Settings::Instance()
{
  static Settings settings;
  return settings;
}
```

would also be equivalent without the need for `std::call_once()`